libfwupdplugin/fu-crc.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2022 Richard Hughes <richard@hughsie.com>
+//
+// SPDX-License-Identifier: LGPL-2.1+
```

Comment:
do we have clang rules already for rust files?  If not, probably a good time to add them.

Reply:
rust has it's own formatter unfortunately. I'll look into setting up precommit.

Reply:
Because we're not using cargo the built-in one fails -- but I've added a tiny python version in contrib/ci